c2corg_api/models/feed.py

```diff
@@ -137,6 +139,10 @@ class DocumentChange(Base):
     activities = Column(
         ArrayOfEnum(enums.activity_type), nullable=False, server_default='{}')
 
+    # enums.default_lang ... OR ... ARRAY(String)
+    lang_ids = Column(
```

Comment:
What about naming this attribute ``langs`` instead? Those are not really ids.
Or ``available_langs``?